performance-tests/README.md

```diff
@@ -31,14 +36,16 @@ For each test we will only note the average duration.
 
 ### Document without an image (`unwrapped_document.json`)
 
-|           | 1 000 | 10 000 | 50 000 | 100 000 | 500 000 | 1 000 000 |
-| --------- | ----- | ------ | ------ | ------- | ------- | --------- |
-| Machine 1 | 2.24s | 14.45s | 67.84s | 184.26s | 898.89s | NA        |
-| Machine 2 | 0.56s | 5.20s  | 25.69s | 57.35s  | 320.14s | 681.89s   |
+|           | 1 000 | 10 000 | 50 000 | 100 000 | 500 000   | 1 000 000  |
+| --------- | ----- | ------ | ------ | ------- | --------- | ---------- |
+| Machine 1 | 2.24s | 14.45s | 67.84s | 184.26s |   898.89s | NA         |
+| Machine 2 | 0.56s | 5.20s  | 25.69s | 57.35s  |   320.14s | 681.89s    |
+| Machine 3 | 1.46s | 15.82s | 77.54s | 151.24s | 1 250.59s | 3 671.40s  |
 
 ### Document with an image (`unwrapped_document_wImage.json`)
 
 |           | 1 000  | 10 000 | 50 000  | 100 000 | 500 000   | 1 000 000 |
 | --------- | ------ | ------ | ------- | ------- | --------- | --------- |
 | Machine 1 | 5.17s  | 32.14s | 170.59s | 338.51s | 2 239.80s | NA        |
 | Machine 2 | 2.48s  | 22.77s | 119.81s | 239.15s | 1 511.12s | NA        |
+| Machine 3 | 2.82s  | 28.80s | 150.77s | 412.93s | 1 250.59s | NA        |
```

Comment:
Your timing for the last test is a bit off compare to the rest. Are you sure about the number?

Reply:
mmmm i can run the ones with image again, probably some background task affecting the numbers

Reply:
the test just finished, i have corrected the time to (2781.98s)